.travis.yml

```diff
@@ -104,6 +108,8 @@ install:
    set +ex
 
 script:
+ - GHC_OPTIONS="-Werror"
+ - if [ $GHCVER = 8.2.1 ]; then GHC_OPTIONS="-Wno-missing-home-modules"; fi
```

Comment:
Should -Werror be included here as well?

Reply:
good catch!